addons/refuel/functions/fnc_makeJerryCan.sqf

```diff
@@ -22,12 +22,15 @@ if (isNull _target ||
     {_target isKindOf "AllVehicles"} ||
     {_target getVariable [QGVAR(jerryCan), false]}) exitWith {};
 
-if (isServer) then {
-    [_target, _fuelAmount] call FUNC(setFuel);  // has global effects
-};
+
 _target setVariable [QGVAR(jerryCan), true];
 _target setVariable [QGVAR(source), _target];
 
+if (isServer) exitWith {
```

Comment:
don't think exitWith is right

Reply:
Nope it isn't, but would irritatingly have worked on dedicated.
Fixed to then.